CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 List of the most important changes for each release.
 
+## 0.13.0 (unreleased)
+
+### Changed or fixed
+
+- Fixed: Unclean shutdowns on very large databases, due to prolonged database cleanup
+
```

Comment:
did the merge work correctly? This shouldn't be in the diff

Reply:
Yeah, I merged develop into this branch after #5888 was merged into develop.